src/notarytool.ts

```diff
@@ -83,24 +83,45 @@ export async function notarizeAndWaitForNotaryTool(opts: NotaryToolStartOptions)
     ];
 
     const result = await spawn('xcrun', notarizeArgs);
-    const parsed = JSON.parse(result.output.trim());
 
-    if (result.code !== 0 || !parsed.status || parsed.status !== 'Accepted') {
+    if (result.code !== 0) {
```

Comment:
Totally a style thing and others may disagree with me. I think the code is easier to follow by using an early return rather than a long `if` block.

```
if (result.code === 0) {
  d('notarization success');
  return
}
```

Reply:
I think so, I just almost never early return a successful result haha.  It felt wrong.